contrib/hadoop/src/main/java/org/xtreemfs/common/clients/hadoop/XtreemFSFileSystem.java

```diff
@@ -84,26 +84,14 @@ public void initialize(URI uri, Configuration conf) throws IOException {
             Logging.logMessage(Logging.LEVEL_DEBUG, this, "init : " + uri);
         }
 
-        String defaultURIString = conf.get("fs.defaultFS");
```

Comment:
It's just that XtreemFS may not always be the default file system, e.g. when you mix sources from HDFS and XtreemFS. In that case, querying fs.defaultFS will give the HDFS URI. I will add a fallback to the URI passed to the constructor (typically the file URI through which XtreemFS is accessed the first time), and if that does not contain the volume then I'll add another fallback to fs.defaultFS.